docs/guides/stop_job.md

```diff
@@ -0,0 +1,42 @@
+!!! important "Quick Look"
+    From the CLI:
+    ```
+    $ floyd stop <username>/<project_name>/<job_number>
+    ```
+
+    From floydhub.com:
+    ![Stop Job](../img/stop_job.jpg)
+
+
+You can stop a queued or running job using Floyd CLI or using the web interface
+on [floydhub.com](https://www.floydhub.com).
+
+## Using the CLI
+A job can be stopped using the `floyd stop` command and passing it the name of
+your project, as shown below:
+
+```
+$ floyd stop mckay/projects/my-project/4
+Experiment shutdown request submitted. Check status to confirm shutdown
+```
+
+To check the status of your job to make sure it shut down properly, use the
```

Comment:
should be `make sure it shuts`?

Reply:
I was intending it to be in the past tense, so this would be correct. But the fact that it stood out to you as reading oddly means I should probably update it anyways. I'll make a change quick

Reply:
updated